### DISCUSSION TITLE
Adding New Menubar to a ReactDialog

### DISCUSSION THREAD

**Discussion by samael-sam** [eclipse-theia/theia#10483] [Q&A] [answered]
Hi,
I wish to add a new Menubar similar like Theia's Menubar to a ReactDialog that takes up full screen. It would be great if any could help me with this.

Thanks.

**Comment by samael-sam** [comment, marked as answer]
Can you assist  @vince-fugnitto  @msujew  ?

**Reply by vince-fugnitto** [reply]
@samael-sam I'd love to help but unfortunately the question is a little vague at the moment, would you mind adding additional details. Are you looking into implementing a separate menubar, or re-creating the existing main menu in the react dialog for some reason. What types of commands do you want present in the menubar?

**Reply by samael-sam** [reply]
Hi @vince-fugnitto ,
Sorry for being vague. I wish to create a new React Dialog with a complete new set of Menu with Menu items that are completely different from the existing Menubar. 

For this example lets assume I wish to have just a Sample MenuItem within File1 menu. 
i.e File1 ~> Sample

**Reply by vince-fugnitto** [reply]
@samael-sam thank you for clarifying!

For the specific use-case you can use whichever implementation you'd like, the framework does not impose one or the other. If it's sufficient you might want to use [react navbars](https://react-bootstrap.github.io/components/navbar/) directly, it would likely be the simplest implementation.

**Reply by samael-sam** [reply]
@vince-fugnitto Thank you for your quick reply. Sure, I will do that then. Thanks.
